vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -189,7 +179,7 @@ private void setComponentStyles() {
   }
 
   public void projectId(String projectId) {
-    handler.projectId(projectId);
+    handler.setProjectId(projectId);
```

Comment:
This was to keep the method naming schema consistent between the components (`ExperimentListComponent`, `ProjectNavigationBarComponent` etc. ...) . 
An alternative would be to name all handler methods `handler.projectId()` instead of `handler.setProjectId(`) if you think that's more explicit 👍